Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
It's a fair point. Mostly to be consistent with the tests for FakeOptionsBuilders and DummyFactories. If you're prefer cooler a TestCaseSource here, with the caveat that the test would turn into an "inconclusive" if we removed the formatters at some time in the future, I'm happy to comply.

Alternatively, I could experiment with a test case source that _if there were no formatters (dummy factories, options builders)_, adds a test-provided source that would pass.